tests/functional/generate_pages.py

```diff
@@ -489,9 +525,10 @@ def write_summary_spec(page_spec, section, collapsible, section_summary):
 
                     page_spec.write(SUMMARY_ANSWER_GETTER.substitute(answer_context))
 
-                    page_spec.write(
-                        SUMMARY_ANSWER_EDIT_GETTER.substitute(answer_context)
-                    )
+                    if summary_type != "view_submitted_response":
+                        page_spec.write(
+                            SUMMARY_ANSWER_EDIT_GETTER.substitute(answer_context)
+                        )
```

Comment:
I think some of this may not have been quite right prior to your changes, but there doesn't appear to be any difference between:

- `SUMMARY_ANSWER_GETTER` and `SECTION_SUMMARY_ANSWER_GETTER` or 
- `SUMMARY_ANSWER_EDIT_GETTER` and `SECTION_SUMMARY_ANSWER_EDIT_GETTER`

It looks like we could remove `SECTION_SUMMARY_ANSWER_GETTER` and `SECTION_SUMMARY_ANSWER_EDIT_GETTER` and simplify this logic.

Reply:
`SUMMARY_ANSWER_GETTER` and `SUMMARY_ANSWER_EDIT_GETTER` are added twice. The only logic that should be needed is to only add `SUMMARY_ANSWER_EDIT_GETTER` if `summary_type != "view_submitted_response"`.

Reply:
@LJBabbage I'm not sure if you've seen my previous comment, but there is a still a bit of duplication in this bit of code that should be removed.

Current:
```python
                    if summary_type == "section":
                        page_spec.write(
                            SUMMARY_ANSWER_GETTER.substitute(answer_context)
                        )
                        page_spec.write(
                            SUMMARY_ANSWER_EDIT_GETTER.substitute(answer_context)
                        )

                    page_spec.write(SUMMARY_ANSWER_GETTER.substitute(answer_context))

                    if summary_type != "view_submitted_response":
                        page_spec.write(
                            SUMMARY_ANSWER_EDIT_GETTER.substitute(answer_context)
                        )
```

Proposed:
```python
                    if summary_type != "view_submitted_response":
                        page_spec.write(
                            SUMMARY_ANSWER_EDIT_GETTER.substitute(answer_context)
                        )

                    page_spec.write(SUMMARY_ANSWER_GETTER.substitute(answer_context))
```

Reply:
Lets try to keep this to the story, happy to do the proposed as Andrew has gone to the effort, but anything above that we should write another story. There was a keenness to get this in and it blocks other cards. I am even happy to do the refactor after, just don't want to go to far down the rabbit hole, not forgetting this wasn't going to have functional tests anyway at this stage

Reply:
Done